pkg/auth/api/user/user_actions_test.go

```diff
@@ -0,0 +1,81 @@
+package user
+
+import (
+	"testing"
+)
+
+func TestValidatePassword(t *testing.T) {
+	tests := []struct {
+		name       string
+		username   string
+		password   string
+		expectsErr bool
+	}{
+		{
+			name:       "password too short",
+			username:   "admin",
+			password:   "admin",
```

Comment:
Ideally, we don't want these equal here - just have a short password as currently it also matches case when username and password match (making it the same case as below)